microsoft.ui.xaml.automation.peers/automationpeer_setparent_93769970.md

```diff
@@ -17,7 +17,7 @@ Sets the [AutomationPeer](automationpeer.md) that is the parent of this [Automat
 The parent automation peer.
 
 ## -remarks
-> You can't re-parent an [AutomationPeer](automationpeer.md) for a UWP app. This functionality is only available for a Windows Phone 8.x app.
+> You can't re-parent an [AutomationPeer](automationpeer.md) for a UWP app. This functionality was only available for a Windows Phone 8.x app.
```

Comment:
We also need to remove UWP from here. These docs are for Windows App SDK / WinUI 3. @Karl-Bridge-Microsoft  - Do you know if this statement is true for App SDK?
```suggestion
> You can't re-parent an [AutomationPeer](automationpeer.md) for a Windows App SDK app.
```

Reply:
This does not appear to be a constraint in the source code. I suggest deleting the entire note.

Reply:
Thanks for checking that, Karl. I'll remove it entirely.